### ISSUE TITLE
Any chance to get rid of jQuery dependency?

### ISSUE THREAD

**Issue by Kopyov** [alizahid/slinky#49]
Do you by any chance consider removing jQuery dependency in the near future?

**Comment by alizahid** [comment]
Yeah, I've been thinking about dropping jQuery and rewriting this with React. Thoughts?

**Comment by Kopyov** [comment]
If you can do vanilla JS version first, that would be extremely useful. To follow current trend where major frameworks are moving towards getting rid of jQuery dependency, having pure JS component will give us all broader range of use cases.
It's a great component and your work is very much appreciated!

**Comment by alizahid** [comment]
Thank you so much! I hope to get on this soon. Build a vanilla JS version and then a React version from it and so on.

**Comment by Kopyov** [comment]
Any progress on this? 😄

**Comment by alizahid** [comment]
Wow, it's been a year and I forgot about it completely. No promises, but maybe one of these days? :smile:

**Comment by teqoit** [comment]
hey, is there any updates?
